lib/migrate_ast.ml

```diff
@@ -109,6 +119,27 @@ module Pparse = struct
     |> to_406.copy_structure
 end
 
+module String = struct
```

Comment:
Could you move these modules to a new `Compat` module or something similar? Not sure why they are inside `Migrate_ast`